frontend/components/ManageStakingPage/StakingContractSection/useMigrate.tsx

```diff
@@ -53,10 +53,10 @@ export const useMigrate = (stakingProgramId: StakingProgramId) => {
 
   const stakingContractInfo = stakingContractInfoRecord?.[stakingProgramId];
 
-  const { hasInitialLoaded: isServicesLoaded } = useServices();
+  const { isLoaded: isServicesLoaded } = useServices();
 
   const minimumOlasRequiredToMigrate = useMemo(
-    () => getMinimumStakedAmountRequired(serviceTemplate, stakingProgramId),
+    () => getMinimumStakedAmountRequired(serviceTemplate, stakingProgramId), // TODO: refactor, can no longer use service template, must use config for funding requirements
```

Comment:
configs referring to the objects in the configs folder in frontend

need to use minimum staked requirements to STAKING_PROGRAMS config instead of the current function; is an old function that held all the staking requirements that never got refactored